### ISSUE TITLE
google_sign_in_web exception with deployed js file

### ISSUE THREAD

**Issue by robsonmeemo** [flutter/flutter#52031]
The google_sign_in_web package works locally when I run my app with `flutter run -d web-server`. But after I `flutter build web` and deploy I get the following exception in chrome:
```
cb=gapi.loaded_0:163 Uncaught TypeError: t.gxp is not a function
    at Se.$1 (google_sign_in_web.dart:113)
    at Object.aom (js_helper.dart:1062)
    at Object.ae9 (core_patch.dart:85)
    at aqG (js_patch.dart:542)
    at js_patch.dart:520
    at e.wo (cb=gapi.loaded_0:175)
    at kk (cb=gapi.loaded_0:178)
    at fk (cb=gapi.loaded_0:178)
    at _.Qj.qU (cb=gapi.loaded_0:178)
    at Nj (cb=gapi.loaded_0:169)
```

**Comment by robsonmeemo** [comment]
So I was able to figure out this was an issue with the authorized urls. But I think the bug is still relevant as the exception that the library tries to throw fails with the error above.

**Comment by klapauciusisgreat** [comment]
Can you elaborate what the issue with authorized urls was ? I seem  to have run into the same issue, but the problem only manifests itself on the firebase deployment on Chrome. no exceptions on Safari or in a local sandbox.

**Comment by robsonmeemo** [comment]
On the google side you have to register which urls are allowed to start the sign-in process. I had a typo in that configuration.

**Comment by klapauciusisgreat** [comment]
Yeah, I had registered them and still got the exception. It turns out you get this exception also when 3rd party cookies are blocked. Really annoying that the exception is so obfucated. And that Google signin requires 3rd party cookies.
